### DISCUSSION TITLE
Webhook

### DISCUSSION THREAD

**Discussion by Ilovejam** [FlowiseAI/Flowise#921] [General]
Hello everyone, I am using Flowise for a while. Is there a way to render webhook code for WhatsApp chatbots that i built in Twilio_

**Comment by chungyau97** [comment]
Hi @Ilovejam, 

The current way to do it is using Zapier/Pipedream to get a trigger and call Flowise API then return the output to original or other location.

Or have a look on the tool https://github.com/FlowiseAI/Flowise/issues/920 that he shared in his video

**Reply by Ilovejam** [reply]
Thank you @chungyau97
